modules/ding_seo/ding_seo.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file
+ * Ding SEO asynchronous JSON-LD.
+ */
+
+(function($) {
+  "use strict";
+
+  // TODO: Add comment explaining why we're not using Drupal behaviors.
+  $(function() {
+    var tingObjectId = Drupal.settings.dingSeo.tingObjectId;
+    $.get(`/ding_seo/jsonld/ting_object/${tingObjectId}`, function(jsonld) {
```

Comment:
Updated comment:

```
  // Bypass Drupal behaviors since we just need to insert a script in header
  // once and have no need to interact with other elements on the page.
```

At least i believe that was the reason :)